rancher2/resource_rancher2_cluster.go

```diff
@@ -545,25 +546,36 @@ func findClusterRegistrationToken(client *managementClient.Client, clusterID str
 				if !IsNotFound(err) {
 					return nil, err
 				}
+				log.Printf("[TRACE] Cluster registration token %s not found for %s", regTokenID, clusterID)
 				break
 			}
 			if (len(regToken.Command) > 0 && len(regToken.NodeCommand) > 0) || retry == retries {
+				log.Printf("[INFO] Found existing cluster registration token for %s", clusterID)
 				return regToken, nil
 			}
+			log.Printf("[DEBUG] Sleeping for 3 seconds  cluster registration token for %s", clusterID)
```

Comment:
reads weird, maybe `[DEBUG] Sleeping for 3 seconds before checking cluster registration token for %s`?